src/paho/mqtt/publish.py

```diff
@@ -19,53 +19,45 @@
 broker, then disconnect and nothing else is required.
 """
 
+import ssl
```

Comment:
You're right, I was working on master branch, so I needed to import it. Now it is not needed anymore.